.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,211 @@
+name: Bug report
+description: File a bug report
+title: "[BUG] "
+labels:
+  - kind/bug
+  - meta/needs triage
+  - priority/medium
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for participating in this project and reporting a bug. Docker Mail Server (DMS) is a community-driven project, and each contribution counts.
+
+        Please **fill out all the fields and checkboxes of this form** to make it easier for maintainers to understand the problem and to solve it. The maintainers and moderators are volunteers that need you to fill this template with accurate informations in order to help you in the best and quickest way. We will have to label your request with `meta/no template - no support` if your request is sloppy and provides no way to help you correctly.
```

Comment:
```suggestion
        Please **fill out all the fields and checkboxes of this form** to make it easier for maintainers to understand the problem and to solve it.
        The maintainers and moderators are volunteers that need you to fill this template with accurate informations in order to help you in the best and quickest way.
        We will have to label your request with `meta/no template - no support` if your request is sloppy and provides no way to help you correctly.
```

Reply:
I thought that this information belongs together and should therefore be "on the same line".

Reply:
Oh okay, I am not sure where I can see a GUI of this config

Reply:
Sadly, no one can. I'd like that too. We can always put lines in between, but I'd start off by trying to put all this in one line :D

Reply:
"informations" -> "information"